ledger-core/Makefile.testing

```diff
@@ -25,7 +25,6 @@ test_with_coverage: ## run unit and slow tests with generation of coverage file
 
 .PHONY: test_func
 test_func: ## run functest FUNCTEST_COUNT times
-	pkill -9 insolard || echo "No insolard running" # dirty hack to terminate all insolard processes
```

Comment:
it helped a lot =(

Reply:
I moved it to launchnet.sh where it should be